core/src/main/java/org/sonarsource/sonarlint/core/container/analysis/SonarLintPathPattern.java

```diff
@@ -60,7 +60,7 @@ public boolean match(InputFile inputFile) {
   }
 
   public boolean match(InputFile inputFile, boolean caseSensitiveFileExtension) {
-    String path = PathUtils.sanitize(inputFile.uri().getPath());
+    String path = PathUtils.sanitize(inputFile.relativePath());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "relativePath"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)